### PR TITLE
Set version to 0.8.25

### PR DESCRIPTION
**Pull request by nikola-matic**
~~Base branch should be changed to develop once https://github.com/ethereum/solidity/pull/14810 is merged.~~

### CODE REVIEW

**Review by cameel — APPROVED** [review]

File: Changelog.md
Comment (badnewz80) [inline Changelog.md:1]: 👍

**Review by badnewz80 — COMMENTED** [review]
👍

**Review by osarukun — COMMENTED** [review]
no

**Comment by badnewz80** [comment]
What should I do?

File: Changelog.md
Comment (badnewz80) [reply Changelog.md:1]: Review completed

**Pull request merged**